common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala

```diff
@@ -68,7 +68,8 @@ class CouchDbRestClient(protocol: String, host: String, port: Int, username: Str
     // Additional queue in case all connections are busy. Should hardly ever be
     // filled in practice but can be useful, e.g., in tests starting many
     // asynchronous requests in a very short period of time.
-    private val requestQueue = Source.queue(128, OverflowStrategy.dropNew)
+    private val QUEUE_SIZE = 16* 1024;
```

Comment:
I suppose there's a few other similar values that if we were to pull out we should do consistently. So considered and rejected at this point. I can't think of a good reason one would change this alone at deployment.